flags.html

```diff
@@ -9,7 +9,9 @@
     <div class="white halfRectangle"></div>
 </div>
 <div id="columbia" class="flagContainer">
-
+    <div class="mustardYellow halfRectangle"></div>
```

Comment:
while this works, you could have done it without the additional class:

```
<div class = "mustardYellow halfRectangle"></div>
<div class = "halfRectangle"></div>
   <div class="halfRectangle darkBlue"></div>
   <div class="halfRectangle darkRed"></div>
</div>
```

Reply:
that way you can conceptually keep similar pieces together.  Two half pieces, with 2 half pieces inside one of the half pieces.